wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/MainActivity.kt

```diff
@@ -173,20 +176,28 @@ fun WearApp(
         }
 
         scrollable(
-            route = PodcastsScreen.route,
+            route = PodcastsScreen.routeHomeFolder,
         ) {
-            NowPlayingPager(
+            PlayerAndPodcastsScreen(
                 navController = navController,
                 swipeToDismissState = swipeToDismissState,
                 scrollableScaffoldContext = it,
-            ) {
-                PodcastsScreen(
-                    listState = it.scrollableState,
-                    navigateToPodcast = { podcastUuid ->
-                        navController.navigate(PodcastScreen.navigateRoute(podcastUuid))
-                    }
-                )
-            }
+            )
+        }
+
+        scrollable(
+            route = PodcastsScreen.routeFolder,
+            arguments = listOf(
+                navArgument(PodcastsScreen.argumentFolderUuid) {
+                    type = NavType.StringType
+                }
+            ),
+        ) {
+            PlayerAndPodcastsScreen(
+                navController = navController,
+                swipeToDismissState = swipeToDismissState,
+                scrollableScaffoldContext = it,
+            )
```

Comment:
I had trouble adding both the route for the home folder and after you tap into a folder, so I just moved the content into a separate reusable component.